mods/butterflies/init.lua

```diff
@@ -57,6 +57,12 @@ for i in ipairs (butter_list) do
 				minetest.set_node(pos, {name = "butterflies:hidden_butterfly_"..name})
 			end
 			minetest.get_node_timer(pos):start(30)
+		end,
+		on_rightclick = function(...)
+			return false
```

Comment:
This should either return an itemstack or nil. Why `false`? Is this callback registration necessary at all?

Reply:
The callback is necessary, otherwise you would still be able to place nodes using the butterfly. Sorry about the return value, i fixed it in another commit.

Reply:
Well, that's still possible using shift+place. Also a comment would be more appropriate to avoid this being reverted in a "cleanup PR".

Reply:
Thanks for the hint. I didn't know about the Shift + RMB thing. I have searched lua_api.txt on how to intercept this combination but i had no luck. How would one make this work? Seems like there is no callback to use.

You're right about the comment, i'll add them once i solved the shift click problem.

Reply:
Nodes can be placed as long the butterfly has a selectionbox. I don't think there's any workaround yet, aside converting the butterfly to an object (which is far from ideal).

Reply:
> I don't think there's any workaround yet

Mmh, too bad. Should i close this pull request or would it make sense to merge it in the current state nevertheless?